backend-api/src/functions/jwks/jwksBuilder/tests/jwksBuilder.test.ts

```diff
@@ -129,40 +170,70 @@ describe("JWKS Builder", () => {
-        const mockPublicKey: Buffer = createPublicKey({
+    describe("Given the KMS public keys can be successfully formatted as JWKs", () => {
+      it("Returns the JWKS with both encryption and signing keys", async () => {
+        const mockEncryptionKey: Buffer = createPublicKey({
           key: {
             kty: "RSA",
             n: "kOBby1nEUcKc-94zIa2qCyqDSE1-2bLWkVjeF3DWY_0v2j9wlLSaR6asONen_HP40wftLOSPYRcKYv6Cjz3LOY7aQYznX14EXSgJxrDwQ7AleX2VS_HB34LMZEa3xmSSH7pLtw_vmJgCNss0zDQLCz1sQwZxlqphF18FdTTUrXbJ9Qk3xIrEzvL2naO2r6WoLBQ9tSr2Sz9TTcJQptfh6hOAHm66oPA6F9uCmbTDEQeI-wLiMMArtcKrGiPAFluo8f0qNkzLRMFIqyadnZ9OZ5u0-H_urOkmLJ2nbAnyTcO-9QeDlomdEMz3yEaJeUoq-jnPpVEfIbd8-07fl7M27w",
             e: "AQAB",
             use: "enc",
             alg: "RSA-OAEP-256",
-            kid: "da48d440-8e51-4383-9a3a-b91ce5adcf2a",
+            kid: "test-key-id",
           },
           format: "jwk",
         }).export({ format: "der", type: "spki" });
-        mockKmsClient.on(GetPublicKeyCommand).resolves({
+
+        const mockSigningKey: Buffer = createPublicKey({
+          key: {
+            kty: "EC",
+            x: "ZvtYNUEFSfoivixzC76PPJk-ka7pvAaidUiZpaHznC4",
+            y: "01kuST7C4NRZlIPpBvuSrrRe9jyWfQtclSBNOv20y94",
+            crv: "P-256",
+            use: "sig",
+            alg: "ES256",
+            kid: "verifiable-credential-signing-key-id",
+          },
+          format: "jwk",
+        }).export({ format: "der", type: "spki" });
+
+        mockKmsClient.on(GetPublicKeyCommand, { KeyId: keyIds[0] }).resolves({
           KeyUsage: "ENCRYPT_DECRYPT",
           KeySpec: "RSA_2048",
-          PublicKey: new Uint8Array(mockPublicKey),
-        } as GetPublicKeyCommandOutput);
+          PublicKey: new Uint8Array(mockEncryptionKey),
+        });
+
+        mockKmsClient.on(GetPublicKeyCommand, { KeyId: keyIds[1] }).resolves({
+          KeyUsage: "SIGN_VERIFY",
+          KeySpec: "ECC_NIST_P256",
+          PublicKey: new Uint8Array(mockSigningKey),
+        });
 
         const buildJwksResponse = await jwksBuilder.buildJwks();
 
         expect(buildJwksResponse.isError).toBe(false);
-        expect(buildJwksResponse.value).toStrictEqual({
-          keys: [
-            {
-              alg: "RSA-OAEP-256",
-              e: "AQAB",
-              kid: keyId,
-              kty: "RSA",
-              n: "kOBby1nEUcKc-94zIa2qCyqDSE1-2bLWkVjeF3DWY_0v2j9wlLSaR6asONen_HP40wftLOSPYRcKYv6Cjz3LOY7aQYznX14EXSgJxrDwQ7AleX2VS_HB34LMZEa3xmSSH7pLtw_vmJgCNss0zDQLCz1sQwZxlqphF18FdTTUrXbJ9Qk3xIrEzvL2naO2r6WoLBQ9tSr2Sz9TTcJQptfh6hOAHm66oPA6F9uCmbTDEQeI-wLiMMArtcKrGiPAFluo8f0qNkzLRMFIqyadnZ9OZ5u0-H_urOkmLJ2nbAnyTcO-9QeDlomdEMz3yEaJeUoq-jnPpVEfIbd8-07fl7M27w",
-              use: "enc",
-            },
-          ],
-        });
+        if (!buildJwksResponse.isError) {
+          expect(buildJwksResponse.value.keys.length).toBe(2);
+
+          expect(buildJwksResponse.value.keys[0]).toMatchObject({
```

Comment:
Nitpick: we could extract this into a const to avoid reaching into the object three times.. Ditto for the signing key.
```suggestion
          const encryptionKey = buildJwksResponse.value.keys[0]
          expect(encryptionKey).toMatchObject({
```